README.md

```diff
@@ -229,32 +229,26 @@ Notice that we have to pass the option `-s` to `pytest` so that it shows what we
 The maze is a grid. Each square in the grid is defined by its coordinates. The default width of the maze is `32` squares, the default height is `16` squares. The coordinate system has the origin `(0, 0)` in the top left (North-West) of the maze and its maximum value `(31, 15)` in the bottom right (South-East). Each square which is not a wall can be empty or contain a food pellet or one or more bots. The different mazes are called `layouts`. For the tournament all layouts will have the default values for width and height and will have a wall on all squares around the border.
 
 ### The `move` function
-**`move(bot, state) ⟶ (x, y), state`**
+**`move(bot, state) ⟶ (x, y)`**
 
 The `move` function gets two input arguments:
 
 - **`bot`** is a reference to the bot in your team corresponding to the current turn. It is an instance of the [`Bot` object](#the-bot-object), which contains all information about the current state of the game
 
-- **`state`** is an arbitrary object, `None` by default, which can be used to hold state between rounds. Example of usage for `state` can be found in [demo04_basic_attacker.py](demo04_basic_attacker.py), [demo05_basic_defender.py](demo05_basic_defender.py), [demo06_one_and_one.py](demo06_one_and_one.py):
+- **`state`** is a place which can be used to hold state between rounds. It is initialized as an empty dictionary when the game starts, and the `move` function can store whatever it wants in it.
+
+Example of usage for `state` can be found in [demo04_basic_attacker.py](demo04_basic_attacker.py), [demo05_basic_defender.py](demo05_basic_defender.py), [demo06_one_and_one.py](demo06_one_and_one.py):
     ```python
     def move(bot, state):
-        if state is None:
-            # initialize an empty dictionary to keep information we
-            # want to share within our team during the game
-            # -> both bots will have access to this dictionary
-            state = {}
-        ...
-        return next_pos, state
+        assert isinstance(state, dict)
```

Comment:
what? why `isinstance`?

Reply:
This is just to underline the fact that this is a dictionary.

Reply:
I'll remove it if it is confusing.

Reply:
> This is just to underline the fact that this is a dictionary.

as a result we will have students putting `isinstance` all over the place thinking that this is some kind of magic, especially given that `assert` is used by `pytest`. I'd say get rid of the check and just leave it to the docs to make it clear.